helper.fish

```diff
@@ -69,6 +69,39 @@ function makeOff ; set -gx SKIP_MAKE On  ; end
 function makeOn  ; set -gx SKIP_MAKE Off ; end
 makeOn
 
+function packageStripNone          ; set -gx PACKAGE_STRIP None    ; end
+function packageStripExceptArangod ; set -gx PACKAGE_STRIP ExceptArangod ; end
+function packageStripAll           ; set -gx PACKAGE_STRIP All     ; end
+packageStripAll
+
+function findMinimalDebugInfo
+  set -l f "$WORKDIR/work/ArangoDB/VERSIONS"
+  set -l MINIMAL_DEBUG_INFO ""
+
+  test -f $f
+  and begin
+    set -l v (fgrep MINIMAL_DEBUG_INFO $f | awk '{print $2}' | tr -d '"' | tr -d "'")
+
+    if test "$v" = "" -o "$v" = "Off" -o "$v" != "On"
+      set MINIMAL_DEBUG_INFO Off
+    else
+      set MINIMAL_DEBUG_INFO On
+    end
+  end
+  or begin
+    set MINIMAL_DEBUG_INFO Off
+  end
+
+  echo $MINIMAL_DEBUG_INFO
```

Comment:
Will this just print `On`, `Off` or an empty string without any context? If so, I suggest we remove the log message here.

Reply:
It will always print either "On" or "Off". Since fish doesn't have return of string it's the only way to use this in `test (findMinimalDebugInfo) = "On"` afterwards.

Reply:
Ah ok, I didn't know that.